src/css.js

```diff
@@ -52,3 +82,38 @@ if ( jQueryVersionSince( "3.4.0" ) && typeof Proxy !== "undefined" ) {
 		}
 	} );
 }
+
+if ( !jQuery.cssNumber ) {
+	jQuery.cssNumber = {};
+}
+
+// jQuery 3.x uses jQuery.cssNumber internally so we can't warn on access there.
+if ( jQueryVersionSince( "4.0.0" ) ) {
+	migrateWarnProp( jQuery, "cssNumber", jQuery.cssNumber,
+		"jQuery.cssNumber is deprecated" );
+}
```

Comment:
@dmethvin I wonder if this warning is useful even for jQuery 4.x. People may still need conditional code adding more props to that object and then they'd hit the warning even if they guard access with an `if` statement. It seems impossible to warn only on invalid usage. Should we remove it?

Reply:
I agree, let's remove it.